api/catalog/api/examples/audio_responses.py

```diff
@@ -81,10 +81,10 @@
 audio_detail_200_example = {
     "application/json": base_audio
     | {
-        "attribution": '"Wish You Were Here" by The.madpix.project is licensed under CC-BY-NC-SA 3.0. To view a copy of this license, visit https://creativecommons.org/licenses/by-nc-sa/3.0/.',  # noqa
+        "attribution": '"Wish You Were Here" by The.madpix.project is licensed under CC-BY-NC-SA 3.0. To view a copy of this license, visit https://creativecommons.org/licenses/by-nc-sa/3.0/.',  # noqa: E501
         "audio_set": {
             "title": "Wish You Were Here",
-            "foreign_landing_url": "https://www.jamendo.com/album/145774/wish-you-were-here",  # noqa
+            "foreign_landing_url": "https://www.jamendo.com/album/145774/wish-you-were-here",  # noqa: E501
```

Comment:
We could also extract these long strings (urls, titles) as constants to make the string take up fewer characters...

Reply:
Since the tests cases change every time the sample data is updated, it's best to not use variables and complex Python structures for them. Use of simple dicts with all the data in one place makes them easier to update.